dev_guide/compute_resources.adoc

```diff
@@ -70,27 +229,48 @@ A compute resource is classified with a quality of service based on the specifie
 |Provided when a request and limit are not specified.
 
 |*Burstable*
-|Provided when a request is specified that is less than an optionally specified limit.
+|Provided when a request is specified that is less than an optionally specified
+limit.
 
 |*Guaranteed*
-|Provided when a limit is specified that is equal to an optionally specified request.
+|Provided when a limit is specified that is equal to an optionally specified
+request.
 |===
 
-A container may have a different quality of service for each compute resource. For example, a container can have *Burstable* CPU and *Guaranteed* memory qualities of service.  
+A container may have a different quality of service for each compute resource.
+For example, a container can have *Burstable* CPU and *Guaranteed* memory
+qualities of service.
 
-The quality of service has an impact on what happens if the resource is compressible or not. CPU is a compressible resource, whereas memory is an incompressible resource.
+The quality of service has an impact on what happens if the resource is
+compressible or not. CPU is a compressible resource, whereas memory is an
+incompressible resource.
 
 With CPU Resources: ::
-- A *BestEffort* CPU container is able to consume as much CPU as is available on a node with the lowest priority.
-- A *Burstable* CPU container is guaranteed to get the minimum amount of CPU requested, but it may or may not get additional CPU time. Excess CPU resources are distributed based on the amount requested across all containers on the node.
-- A *Guaranteed* CPU container is guaranteed to get the amount requested and no more, even if there are additional CPU cycles available. This provides a consistent level of performance independent of other activity on the node.
+- A *BestEffort* CPU container is able to consume as much CPU as is available on a
```

Comment:
My understanding is that "A *BestEffort* CPU container" means "A container with a *BestEffort* QOS".  The former is obviously more concise (good), but not quickly grokkable (bad).  Hmm...